### DISCUSSION TITLE
似乎是VS和MAUI的一个bug导致了最近的 VS无法编译1Remote，附带解决方案，有空请看看

### DISCUSSION THREAD

**Discussion by Dunkelicht** [1Remote/1Remote#724] [_Development]
感谢1Remote项目！

我正在学习C#，对C#本身并不熟悉。 我尝试在win10下使用VS2022 编译，可惜没有成功，

经过一些搜索，发现可能是MAUI的问题，

在csproj中加入以下内容

<PropertyGroup>
	<WindowsSdkPackageVersion>10.0.19041.34</WindowsSdkPackageVersion>
</PropertyGroup>

恢复nuget后，重新编译可以编译通过。

大家是否也会有类似问题？

来源：

https://github.com/dotnet/maui/issues/25160

https://github.com/dotnet/maui/issues/25150#issuecomment-2404428189

**Comment by VShawn** [comment]
然而本项目并没有使用 MAUI 啊。你可以把编译日志贴出来
